internal/boxcli/shellenv.go

```diff
@@ -74,7 +81,7 @@ func shellEnvFunc(cmd *cobra.Command, flags shellEnvCmdFlags) (string, error) {
 		}
 	}
 
-	envStr, err := box.PrintEnv(cmd.Context(), flags.runInitHook)
+	envStr, err := box.PrintEnv(cmd.Context(), flags.runInitHook, flags.omitWrappersFromPath)
```

Comment:
When adding a second boolean is a good time to switch to a struct :P

Reply:
ah, I knew you'd say this :p